apps/server/src/modules/fwu-learning-contents/fwu-learning-contents-test.module.ts

```diff
@@ -10,6 +10,7 @@ import { S3Client } from '@aws-sdk/client-s3';
 import { LoggerModule } from '@src/core/logger';
 import { AuthenticationModule } from '@src/modules/authentication/authentication.module';
 import { AuthorizationModule } from '@src/modules/authorization';
+import { S3ClientAdapter } from '../files-storage/client/s3-client.adapter';
```

Comment:
@SevenWaysDP we must move this after the merge, to keep the boundarys save for the technical moduls.